addons/l10n_ro_edi_stock/models/stock_picking.py

```diff
@@ -929,3 +929,17 @@ def _l10n_ro_edi_stock_report_unhandled_document_state(self, state: str):
         """
         self.ensure_one()
         self.message_post(body=_("Unhandled eTransport document state: %(state)s", state=state))
+
+    @api.model
+    def _l10n_ro_edi_compute_amount(self, move):
+        """
+        Computes the amount of a stock.move based on the product's list price and quantity.
+        This is used to calculate the 'valoareLeiFaraTva' field in the eTransport document."""
+        product = move.product_id
+        currency = product.currency_id
+        res = product.taxes_id.filtered(lambda t: t.company_id == self.env.company).compute_all(
+            product.list_price, product=product, partner=self.env['res.partner']
+        )
```

Comment:
The `product` and `partner` are not really required - they're for tags and language - neither of them are relevant here.
However we could use @aboo-odoo 's suggestion to do the multiplication and rounding by providing the `currency` and `quantity` values.

Please use named parameters.

>   meaning it needs to be passed to the method as parameter
`move.company_id` should do the trick